From 9248113fde4241f35e82c3d67dc1d8f7e1e27664 Mon Sep 17 00:00:00 2001
From: "Gopal, Puunithaaraj" <puunithaaraj.gopal@intel.com>
Date: Fri, 30 Mar 2018 14:01:24 +0800
Subject: [PATCH 1030/1240] crlmodule-lite: Adding config dependencies between
 crlmodule & crlmodule-lite

When crlmodule is turned on, disable crlmodule-lite.

Change-Id: I116b774a66f449dd5022b25f1034ccaad4eaad5f
Signed-off-by: Gopal, Puunithaaraj <puunithaaraj.gopal@intel.com>
---
 drivers/media/i2c/crlmodule-lite/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/crlmodule-lite/Kconfig b/drivers/media/i2c/crlmodule-lite/Kconfig
index 0b3ff56..5f6b506 100644
--- a/drivers/media/i2c/crlmodule-lite/Kconfig
+++ b/drivers/media/i2c/crlmodule-lite/Kconfig
@@ -2,5 +2,6 @@ config VIDEO_CRLMODULE_LITE
 	tristate "CRL Module sensor support for ICI driver"
 	depends on I2C
 	depends on VIDEO_INTEL_ICI
+    depends on !VIDEO_CRLMODULE
 	---help---
 	  This is a generic driver for CRL based camera modules.
-- 
2.7.4

